distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,27 @@
+// overview - ~ a paragraph
+
+=== ${ddf-registry} Prerequisites
+
+To use the ${ddf-registry}, the following apps/features must be installed:
+
+// a list
+* ${ddf-catalog}
+* ${ddf-platform}
+
```

Comment:
spatial is the real dependency here not sure if typically document the transitive ones.